app/assets/stylesheets/_editor.scss

```diff
@@ -39,6 +39,29 @@
     border-bottom: 1px solid $lighter;
     padding-bottom: 5px;
   }
+
+  .headeranchor {
+    display: none;
+    text-decoration: none;
+  }
+
+  h1:hover, h2:hover, h3:hover, {
+    .headeranchor {
```

Comment:
Please move it into single line selector like `h1:hover, h2:hover, h3:hover > .headeranchor`